src/components/tx/AdvancedParamsForm/index.tsx

```diff
@@ -164,6 +193,13 @@ const AdvancedParamsForm = (props: AdvancedParamsFormProps) => {
                     ></TextField>
                   </FormControl>
                 </Grid>
+              </>
+            )}
+
+            {/* Gas price */}
+            {props.isExecution && (
+              <>
+                <Grid item xs={12} sx={{ padding: '0 !important' }} />
```

Comment:
Do we need this empty element? Deleting it in the DOM doesn't seem to make a difference

Reply:
Ah, not needed anymore. 👍 I made the grid constant in another place.